app/src/main/java/com/yeonae/chamelezone/view/mypage/mycourse/MyCourseActivity.kt

```diff
@@ -10,16 +11,38 @@ import com.yeonae.chamelezone.R
 import com.yeonae.chamelezone.data.model.MyCourseItem
 import com.yeonae.chamelezone.network.room.entity.UserEntity
 import com.yeonae.chamelezone.view.course.CourseDetailActivity
+import com.yeonae.chamelezone.view.course.CourseModifyActivity
 import com.yeonae.chamelezone.view.mypage.MoreButtonFragment
 import com.yeonae.chamelezone.view.mypage.mycourse.adapter.MyCourseRvAdapter
 import com.yeonae.chamelezone.view.mypage.mycourse.presenter.MyCourseContract
 import com.yeonae.chamelezone.view.mypage.mycourse.presenter.MyCoursePresenter
 import kotlinx.android.synthetic.main.activity_my_course.*
 
-class MyCourseActivity : AppCompatActivity(), MyCourseContract.View {
+class MyCourseActivity : AppCompatActivity(), MyCourseContract.View,
+    MoreButtonFragment.OnModifyClickListener, MoreButtonFragment.OnDeleteClickListener {
     private val myCourseRvAdapter = MyCourseRvAdapter()
     override lateinit var presenter: MyCourseContract.Presenter
     var memberNumber: Int = 0
+    lateinit var courseItem: MyCourseItem
+
+    override fun showDeleteResult(response: Boolean) {
+        if(response){
+            if(response){
+                Log.d("courseDelete", response.toString())
+            }
+        }
+    }
+
+    override fun onModifyClick() {
+        val intent = Intent(this, CourseModifyActivity::class.java)
+        intent.putExtra("courseNumber", courseItem.courseNumber)
+        startActivity(intent)
+    }
```

Comment:
```suggestion
    override fun onModifyClick() {
           showCourseModifyActivity()
    }
    
    private fun showCourseModifyActivity(){
            val intent = Intent(this, CourseModifyActivity::class.java)
            intent.putExtra("courseNumber", courseItem.courseNumber)
            startActivity(intent)
        }
```
onMoveClick이 호출되었을때, Course수정 페이지로 간다는 의미를 쉽게 전달할 수 있도록,
함수를 분리하는 것을 권유드려봅니다.

Reply:
수정했습니다
34c894e